src/blocks/social-profiles/test/social-profiles.cypress.js

```diff
@@ -316,6 +317,7 @@ describe( 'Test CoBlocks Social Profiles Block', function() {
 	/**
 	 * Test the coblocks social profiles button size.
 	 */
+	// Same problem as share button
```

Comment:
Feel free to remove the comment if you are not able to reproduce the failing test.